juju/juju
# refactor: remove caas operator code from uniter

This removes all the caas podspec operator code from the uniter.

## QA steps

- Bootstrap (k8s|lxd).
- Deploy stuff.
- Check units are all good!

## Documentation changes

N/A

## Links

**Jira card:** JUJU-
